dpe/src/commands/derive_context.rs

```diff
@@ -196,6 +211,8 @@ impl CommandExecution for DeriveContextCmd {
             || (!dpe.support.internal_dice() && self.uses_internal_dice_input())
             || (!dpe.support.retain_parent_context() && self.retains_parent())
             || (!dpe.support.x509() && self.allows_x509())
+            || (!dpe.support.cdi_export()
+                && (self.allows_create_certificate() && self.allows_exports_cdi()))
```

Comment:
Should this be `(self.allows_create_certificate() || self.allows_exports_cdi())`?

Reply:
From the spec:

> If a DPE supports certificates, the DPE MUST require that the create-certificate argument is true and return an invalid-argument error if it is false

That is the argument section for `export-cdi`.

Reply:
Right, but if `!dpe.support.cdi_export()`, I think we want to fail if _either_ of the flags are present. Right now I think it will only fail if both flags are set.

Reply:
Updated!